.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    if: github.repository == 'ExpediaGroup/dropwizard-template-config'
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Java 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+          server-id: ossrh
+          server-username: SONATYPE_USERNAME # env variable for username in deploy
+          server-password: SONATYPE_PASSWORD # env variable for token in deploy
+          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }} # Value of the GPG private key to import
+          gpg-passphrase: GPG_PASSPHRASE # env variable for GPG private key passphrase
+
+      - name: Build library
+        run: mvn clean build
+
+      - name: Publish library
+        run: |
+          NEW_VERSION=$(echo "${GITHUB_REF}" | cut -d "/" -f3)
+          echo "New version: ${NEW_VERSION}"
+          mvn org.codehaus.mojo:versions-maven-plugin:2.7:set -DnewVersion=${NEW_VERSION}
```

Comment:
instead of `mvn` directly should we use the wrapper instead? (this will ensure the correct version of maven)